src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -1022,10 +1022,18 @@ public class TraceStatus
             /// <value></value>
             public string? Hostname
             {
-                get => _status.Destination != IPAddress.Any.ToString()
-                    && _status.Destination != IPAddress.IPv6Any.ToString()
-                        ? _status.Destination
-                        : null;
+                get
+                {
+                    if (_status.Address?.ToString() == IPAddress.Any.ToString()
+                        || _status.Address?.ToString() == IPAddress.IPv6Any.ToString())
```

Comment:
We could remove ToString() everywhere.

Reply:
Hmm. I'm sure I tried to do a straight equality a while back, and it didn't work for some reason.

It does look like they have a proper Equals() implementation, so... yeah, I'll change this. Thanks!